### ISSUE TITLE
Electron website fails colour contrast

### ISSUE THREAD

**Issue by amosie** [atom/electron#4891]
It's important that all users are able to read a website with ease be it when being viewed by someone having colour deficits or when viewed on a black and white screen.

We should be aiming for all colours to be at least [AA compliant](https://www.w3.org/TR/AERT#color-contrast). 

Unfortunately the Electron website doesn't pass AA compliance in a lot of areas- this means that the contrast is not enough for users with vision problems to be able to view the text easily. 

**Hero area**
![screen shot 2016-03-23 at 14 16 41](https://cloud.githubusercontent.com/assets/2235325/13988003/08b5cef2-f102-11e5-865b-7076a3b0a9e5.png)

**Links in hero area**
![screen shot 2016-03-23 at 14 18 07](https://cloud.githubusercontent.com/assets/2235325/13988038/2269d76c-f102-11e5-9282-3948ca51a4f8.png)

**Links on white**
![screen shot 2016-03-23 at 14 19 04](https://cloud.githubusercontent.com/assets/2235325/13988072/40acb708-f102-11e5-9bf6-7bbc13c9d633.png)

**Top bar navigation**
![screen shot 2016-03-23 at 14 20 34](https://cloud.githubusercontent.com/assets/2235325/13988119/70d3a3ec-f102-11e5-84ee-e7bc619413f7.png)

**More features area**
![screen shot 2016-03-23 at 14 21 30](https://cloud.githubusercontent.com/assets/2235325/13988156/998be362-f102-11e5-9ef8-9daa9a6019b5.png)

All text should pass AA compliance for it to be good enough for everyone to use. If you'd like me to talk through this in more detail I'd be happy to.

cc @simurai @mdo @connors @jlord

**Comment by connors** [comment]
I believe @simurai is in the process of redesigning this whole thing anyway. Is that right, Simon?

**Comment by kevinsawicki** [comment]
@amosie Thanks for raising this issue, would you mind moving it to the repository for the website? https://github.com/atom/electron.atom.io

**Comment by amosie** [comment]
@kevinsawicki eeek sorry - done https://github.com/atom/electron.atom.io/issues/191
